server.py

```diff
@@ -47,54 +48,39 @@ def handle():
     try:
         if content == "about":
             return {"content": "Hello! This is BOGO bot! Please type 'help' for help!"}
+        
+        elif content == "all subs":
+            all_data = utils.get_subscribed_users(supabase_client)
+            for users in all_data.data:
+                print(users['zulip_full_name'])
+            return {"content": all_data.data}
```

Comment:
What does this return? Should we be formatting the output to make it readable?

Reply:
oh this wasnt the final product, this is just testing whats being returned cos i was having trouble with the query for a logn time hehe